### ISSUE TITLE
not draw images

### ISSUE THREAD

**Issue by mohammad69h94** [pierluigiferrari/ssd_keras#85]
Hi, 
My problem is that program not draw image and only show Ground truth boxes and Predicted boxes. Like below: and i use Udacity road traffic dataset with 3 labels : car, truck, pedestrian.

Shape of the 'conv4_3_norm_mbox_conf' weights:
kernel:	 (3, 3, 512, 324)
bias:	 (324,)
[0, 3, 8, 1, 81, 84, 89, 82, 162, 165, 170, 163, 243, 246, 251, 244]
Shape of the 'conv4_3_norm_mbox_conf' weights:
kernel:	 (3, 3, 512, 16)
bias:	 (16,)
Using TensorFlow backend.
WARNING:tensorflow:From C:\python3.6.4.64bit\lib\site-packages\keras\backend\tensorflow_backend.py:3148: calling l2_normalize (from tensorflow.python.ops.nn_impl) with dim is deprecated and will be removed in a future version.
Instructions for updating:
dim is deprecated, use axis instead
Model built.
Weights file loaded: C:/python3.6.4.64bit/ssd_keras-master_3/VGG_coco_SSD_300x300_iter_400000_subsampled_8_classes.h5
Number of images in the dataset: 9218
Image: C:/python3.6.4.64bit/ssd_keras-master_3/object-detection-crowdai/1479498971506814516.jpg

Ground truth boxes:
[[  1 151 171 148 162]
 [  1 200 223 145 166]]
Predicted boxes:
   class   conf xmin   ymin   xmax   ymax
[[  1.     0.85 200.68 145.74 223.27 165.34]
 [  1.     0.5  150.98 148.58 173.56 161.22]]
>>> 
[subsampling.txt](https://github.com/pierluigiferrari/ssd_keras/files/1825580/subsampling.txt)

**Comment by pierluigiferrari** [comment]
If you use the Jupyter notebook as is, it does draw the images just as expected.
